background_scripts/commands.coffee

```diff
@@ -67,6 +67,11 @@ Commands =
           when "unmapAll"
             @keyToCommandRegistry = {}
 
+    # Push the key mapping for passNextKey into Settings so that it's available in the front end for insert
+    # mode.
+    Settings.set "passNextKeyKeys",
```

Comment:
Using `Settings` for this feels a bit wrong, since it's a computed value rather than a setting. Seems like this value could be passed in `getCompletionKeysRequest` to be consistent with how we treat our other calculated mapping values.